src/package-hoister.js

```diff
@@ -666,9 +666,9 @@ export default class PackageHoister {
   }
 
   markShallowWorkspaceEntries() {
-    const targetWorkspace = path.basename(this.config.cwd);
```

Comment:
Why do you need the `focusedWorkspaceName` instead of just using `readRootManifest` here?

Reply:
I suppose we could read the manifest everywhere we need the focused package's name. It seemed simpler and faster to do it once up front because otherwise we'd need to do it like 4-5 times.